### ISSUE TITLE
All field generators incorrectly gated behind osmium?

### ISSUE THREAD

**Issue by NoxyNixie** [FTBTeam/FTB-Interactions#1203]
## Bug Report:
All field generators use osmium making them neigh impossible to make in the very early tiers.

The field generators require a form of osmium wire which as far as I can determine (I have not actually progressed that far yet) is not acquirable before going to the end.

The LV field generator is mostly only used in the basic mass fabricator and basic replicator however the MV one is used in a lot of useful items that can be handy (literally) in MV tier and up but become more or less obsolete once you are able to achieve a meaningful amount of osmium. Some of the MV looking tier items that are gated this way are:  Quantum tank, Quantum chest, Handy Bag. All other components of these recipes seem to indicate craft-ability in MV tier possibly early HV. Similar thing happens with the HV field generator.

## Expected Behavior/Suggestions:
Ability to craft said items in mentioned tiers. A different wire type might be better suited for the field generator recipe.

**Comment by dominic-jones** [comment]
I noticed this too, but presumed it was intended. I agree some quests give suggestions of earlier crafting, but I also am finding no problems progressing without it. If the Quantum Tank was earlier, I should would have used it. But then I wouldn't have enchanted Signalum/Enderium Tanks with Holding 4. Maybe progression?

**Comment by Saereth** [comment]
yeah this is intended however if you want to make a concise list of items you feel should be gated sooner than the end edit your original post with the list and I'll review those items specifically

**Comment by NoxyNixie** [comment]
These are the items and blocks I found that I believe shouldn't be gated behind osmium:

- Items using the Field Generator MV:
  - Basic Quantum Tank
  - Basic Quantum Chest
  - Handy Bag
- Items using the Field Generator HV:
  - Advanced Quantum Tank
  - Advanced Quantum Chest
  - Handy Bag (Large)
- Items using the Field Generator EV:
  - Advanced Quantum Tank II
  - Advanced Quantum Chest II

Furthermore I think the Black Hole Tank and Black Hole Unit should be changed to use a higher tier of Field Generator since these two are a order of magnitude larger than the quantum tank and chest crafted with the same tier of Field Generator.

I don't know much beyond this as I've not progressed beyond HV yet and don't know if for instance the Naquadah Reactor should be attainable before or after osmium.

I'm perfectly fine with having all the mass fabricator and replicator stuff gated behind osmium.

I just noticed the Black-Hole Quantum Tank III and Chest III in the IV tier ... these are still far worse than the Industrial foregoing variants found in the EV Tier.

Block | Capacity | Crafted with
--- | --- | ---
Black Hole Tank (Industrial Foregoing) | 2,147,483,000 mB | Field Generator (EV)
Black-Hole Quantum Tank III | 256,000,000 mB | Field Generator (IV)
Black Hole Unit (Industrial Foregoing) | 2,147,483,647 Items | Field Generator (EV)
Black-Hole Quantum Chest III | 16,384,000 Items | Field Generator (IV)

As I still haven't progressed to these tiers yet I don't really know how to re-balance these correctly.

**Comment by dominic-jones** [comment]
If I had these recipes earlier, I would have used them. However, I feel they'd make all other containers obsolete. I'm generally using holding 4 enchanted signalum portable tanks (960 buckets). That's a pretty huge gap to 64000 buckets.

**Comment by Hjaldar** [comment]
At least the Basic Quantum tanks are available earlier than they can be crafted through dungeon loot. I used the ones I looted selectively before I could craft them. I felt like the Handy Bag became craftable very close to the point I found a use for it. YMMV.

I think the crafting progression is mostly fine as it is. The only change I would consider is possibly moving the IF Black Hole Unit items, which are IV tier due to their circuits, later in progression than the lower capacity IV tier Gregtech items, possibly by including the latter as the central item in the crafting recipe of the former. I'm not sure the effort is worth it here though, particularly in a mature pack.
